src/parser/jobs/mnk/Fists.js

```diff
@@ -58,15 +107,105 @@ export default class Fists extends Module {
 			content: <Fragment>
 				Fist buffs are one of your biggest DPS contributors, either directly with <ActionLink {...ACTIONS.FISTS_OF_FIRE} /> or <StatusLink {...STATUSES.GREASED_LIGHTNING_I} /> manipulation with <ActionLink {...ACTIONS.FISTS_OF_EARTH} /> and <ActionLink {...ACTIONS.FISTS_OF_WIND} />.
 			</Fragment>,
-			why: `No Fist buff was active for ${this.parser.formatDuration(this._stanceless)}.`,
+			why: `${this._fistGCDs[STANCELESS]} GCDs had no Fists buff active.`,
 			tiers: STANCELESS_SEVERITY,
-			value: this._stanceless,
+			value: this._fistGCDs[STANCELESS],
+		}))
+
+		// Semi lenient trigger, this assumes RoE is only used during downtime
+		this.suggestions.add(new TieredSuggestion({
```

Comment:
Is it worth having these as seperate suggestions, esp. given the pie is a thing?

Reply:
The situations are different enough I think it's worth highlighting them separately. Staying in Wind is usually a mistake, but Earth can be intentional (even if more damaging to DPS), and no stance is always bad but takes a full GCD to fix usually. By highlighting them individually, the user also can pay attention to the specific instance they do that instead of freaking out about ongoing FoF uptime I guess.